roles/openshift_metering/tasks/main.yaml

```diff
@@ -1,6 +1,30 @@
 ---
-- include_tasks: install.yaml
-  when: openshift_metering_install | default(true) | bool
+- name: Openshift Metering RDS Database setup
+  include_tasks: rds-database-install.yaml
```

Comment:
import_tasks is preferred over include_tasks.

https://docs.ansible.com/ansible/2.5/user_guide/playbooks_reuse_includes.html

Reply:
It's okay to only do it for the ones which don't use loops right?